krystle_talens-fatima.js

```diff
@@ -0,0 +1,124 @@
+//Calculate Cube
+function calculateCube(num) {
+    console.log(num*num*num)
+  }
+  
+  calculateCube(5)
+  
+//Is a Vowel?
+  var vowels = [ "a", "e", "i", "o", "u","y"] 
+  const isAVowel = (letter) => {
+    if (vowels.includes(letter)) {
+      return true
+    } else {
+      return false
+    }
+  }  
+  console.log(isAVowel("a"));
```

Comment:
nice! I would suggest you keep any variables pertinent to the function inside the function. you can declare your `vowels` variable between lines 10 and 11. this is best practice, unless you need that vowels array for another function